modules/apps/questions/questions-web/package.json

```diff
@@ -0,0 +1,31 @@
+{
+	"dependencies": {
+		"@clayui/button": "3.0.3",
+		"@clayui/form": "3.2.1",
+		"@clayui/icon": "3.0.1",
+		"@clayui/label": "3.0.1",
+		"@clayui/link": "3.0.1",
+		"@clayui/loading-indicator": "3.0.0",
+		"@clayui/modal": "3.1.3",
+		"@clayui/navigation-bar": "3.0.4",
+		"@clayui/sticker": "3.0.3",
+		"classnames": "2.2.6",
+		"frontend-js-react-web": "*",
+		"frontend-js-web": "*",
+		"prop-types": "15.7.2",
+		"react": "16.9.0",
+		"react-router-dom": "^5.0.1"
+	},
+	"description": "",
+	"main": "js/index.es.js",
```

Comment:
do you need main 🤔

Reply:
No! :) https://github.com/boton/liferay-portal/pull/123/commits/b349df74b277312278fa5e9a9a43672cdd734fe7